.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -8,12 +8,6 @@ parameters:
   - name: vrTestPackagePath
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27040&commitId=619d8ee5f87103e530fd834c0f739b3689279c42&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Inverted.chromium.png|246| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>